frontend/javascripts/maintenance_banner.tsx

```diff
@@ -144,3 +147,85 @@ export function MaintenanceBanner() {
 
   return null;
 }
+
+export function UpgradeVersionBanner() {
+  const white = "var(--ant-color-text-primary)";
+  const blue = "var(--ant-color-primary)";
+  const UPGRADE_BANNER_STYLE: React.CSSProperties = {
+    position: "absolute",
+    top: 0,
+    left: 0,
+    height: constants.BANNER_HEIGHT,
+    textAlign: "center",
+    backgroundColor: blue,
+    color: white,
+    fontSize: "medium",
+  };
+  const customParseFormat = require("dayjs/plugin/customParseFormat");
+  dayjs.extend(customParseFormat);
+  const currentDate = dayjs();
+
+  const isVersionOutdated = useFetch(
+    async () => {
+      const buildInfo = await getBuildInfo();
+      const commitDateWithoutWeekday = buildInfo.webknossos.commitDate.replace(
+        /(Mon)|(Tue)|(Wed)|(Thu)|(Fri)|(Sat)|(Sun)\w*/,
+        "",
+      );
+      const lastCommitDate = dayjs(commitDateWithoutWeekday, "MMM DD HH:mm:ss YYYY ZZ"); // todo two digit dates? test more once time tracking is merged
```

Comment:
todo

Reply:
https://git-scm.com/docs/git-log
`--date=default is the default format, and is based on ctime(3) output. It shows a single line with three-letter day of the week, three-letter month, day-of-month, hour-minute-seconds in "HH:MM:SS" format, followed by 4-digit year, plus timezone information, unless the local time zone is used, e.g. Thu Jan 1 00:00:00 1970 +0000.`
maybe not